include/tgfx/svg/SVGExporter.h

```diff
@@ -0,0 +1,91 @@
+/////////////////////////////////////////////////////////////////////////////////////////////////
+//
+//  Tencent is pleased to support the open source community by making tgfx available.
+//
+//  Copyright (C) 2024 THL A29 Limited, a Tencent company. All rights reserved.
+//
+//  Licensed under the BSD 3-Clause License (the "License"); you may not use this file except
+//  in compliance with the License. You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/BSD-3-Clause
+//
+//  unless required by applicable law or agreed to in writing, software distributed under the
+//  license is distributed on an "as is" basis, without warranties or conditions of any kind,
+//  either express or implied. see the license for the specific language governing permissions
+//  and limitations under the license.
+//
+/////////////////////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include <memory>
+#include <sstream>
+#include "tgfx/core/Canvas.h"
+#include "tgfx/core/Rect.h"
+#include "tgfx/gpu/Context.h"
+
+namespace tgfx {
+class SVGExportingContext;
+
+struct ExportingOptions {
+  ExportingOptions() = default;
+  ExportingOptions(bool convertTextToPaths, bool prettyXML)
+      : convertTextToPaths(convertTextToPaths), prettyXML(prettyXML) {
+  }
+  /**
+   * Convert text to paths in the exported SVG text, only applicable to outline fonts. Emoji and
```

Comment:
Convert text to paths in the exported SVG. This only applies to fonts with outlines. Fonts without outlines, like emoji and web fonts, will be exported as text.